docs/migration_guide.rst

```diff
@@ -12,6 +12,10 @@ Migration guide
 
 1. Added :class:`NonZeroType` in order to fix parsing ABI which contains Cairo`s `core::zeroable::NonZero <https://github.com/starkware-libs/cairo/blob/a2b9dddeb3212c8d529538454745b27d7a34a6cd/corelib/src/zeroable.cairo#L78>`_
 
+.. currentmodule:: starknet_py.net
+
+2. Added (SNIP-9)[https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-9.md] support to :class:`account.account.Account`. Now it's possible to create a :class:`client_models.Call` for outside execution using :meth:`account.account.Account.sign_outside_execution_call`.
+
```

Comment:
```suggestion
2. Added `SNIP-9 <https://github.com/starknet-io/SNIPs/blob/main/SNIPS/snip-9.md>`_ support to :class:`~starknet_py.net.account.account.Account`. Now it's possible to create a :class:`~starknet_py.net.client_models.Call` for outside execution using :meth:`~starknet_py.net.account.account.Account.sign_outside_execution_call`.
```

Reply:
Oh... Thank you!

Reply:
How do you view those?

Reply:
It fails now. I'm not a guru at writing documentation with sphinx, tbh... will try to fix.